### PR TITLE
Add 'url' field to RegisteredValidatorParticle

### PR DESCRIPTION
**Pull request by flotothemoon**

### CODE REVIEW

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/validators/RegisterValidatorAction.java [hunk 1]
```diff
@@ -31,15 +31,21 @@
  */
 public class RegisterValidatorAction implements Action {
 	private final RadixAddress validator;
+	private final String url;
 
-	public RegisterValidatorAction(RadixAddress validator) {
+	public RegisterValidatorAction(RadixAddress validator, String url) {
 		this.validator = validator;
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/application/translate/validators/RegisterValidatorAction.java:37]: Should validator be null-checked here?

File: radixdlt-java/src/main/java/com/radixdlt/client/atommodel/validators/RegisteredValidatorParticle.java [hunk 2]
```diff
@@ -49,8 +53,13 @@ private RegisteredValidatorParticle() {
 	}
 
 	public RegisteredValidatorParticle(RadixAddress address, long nonce) {
+		this(address, null, nonce);
+	}
+
+	public RegisteredValidatorParticle(RadixAddress address, String url, long nonce) {
 		super(address.euid());
 		this.address = Objects.requireNonNull(address, "address");
```
Comment (clstrfsck) [inline radixdlt-java/src/main/java/com/radixdlt/client/atommodel/validators/RegisteredValidatorParticle.java:61]: Should address be null-checked here?

**Review by clstrfsck — APPROVED** [review]
Some missing null checks, but otherwise good.

File: radixdlt-java/src/main/java/com/radixdlt/client/atommodel/validators/RegisteredValidatorParticle.java (hunk 2 above)
Comment (flotothemoon) [reply radixdlt-java/src/main/java/com/radixdlt/client/atommodel/validators/RegisteredValidatorParticle.java:61]: Not really necessary, but I think it clarifies intent

File: radixdlt-java/src/main/java/com/radixdlt/client/application/translate/validators/RegisterValidatorAction.java (hunk 1 above)
Comment (flotothemoon) [reply radixdlt-java/src/main/java/com/radixdlt/client/application/translate/validators/RegisterValidatorAction.java:37]: Good idea

**Pull request merged**